### PR TITLE
Bug 1532995 - Cover compare replicates view with FE tests

### PR DESCRIPTION
**Pull request by ionutgoldan**

### CODE REVIEW

File: tests/ui/unit/react/replicates_graph_test.jsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+      filters={{ testSuite: TEST_SUITE_NAME, subtest: SUBTEST_NAME }}
+      injGetData={mockedGetData}
+      injGetReplicateData={mockedGetReplicateData}
+    />,
+  );
+
+test('graph with available data displays properly', async () => {
+  const { getByText } = replicatesGraph(
+    mockedGetData,
+    mockedGetReplicateDataWithData,
```
Comment (sarah-clements) [inline tests/ui/unit/react/replicates_graph_test.jsx]: This naming seems a bit odd  :)

File: ui/perfherder/compare/ReplicatesGraph.jsx [hunk 2]
```diff
@@ -56,16 +57,22 @@ export default class ReplicatesGraph extends React.Component {
...
           height: 275,
         };
 
   fetchReplicateGraphData = async () => {
-    const { projectName, revision, subtestSignature } = this.props;
+    const {
+      projectName,
+      revision,
+      subtestSignature,
+      injGetData,
```
Comment (sarah-clements) [inline ui/perfherder/compare/ReplicatesGraph.jsx]: what does the `inj` mean?

File: tests/ui/unit/react/replicates_graph_test.jsx (hunk 1 above)
Comment (ionutgoldan) [reply tests/ui/unit/react/replicates_graph_test.jsx]: Yeah, I should address this.

File: ui/perfherder/compare/ReplicatesGraph.jsx (hunk 2 above)
Comment (ionutgoldan) [reply ui/perfherder/compare/ReplicatesGraph.jsx]: > edit: I saw your note at the bottom about dependency injection. You could also just pass `getData` and `PerfSeriesModel.getReplicateData` in as props instead (and still including a note would be helpful)

I'll follow this first advice.

**Comment by ionutgoldan** [comment]
Thank you!

**Pull request merged**